src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -462,6 +462,14 @@ impl TargetInfo {
                     // preserved.
                     should_replace_hyphens: true,
                 })
+            } else {
+                ret.push(FileType {
+                    suffix: format!("{}.dwp", suffix),
```

Comment:
.so, or .exe/.dll on mingw

Reply:
I've left a comment about that here, and changed this to use the implicit argument.

I did *not* change the format! I added in the implementation of cargo clean since the surrounding code doesn't use implicit arguments.